src/index.html

```diff
@@ -6,8 +6,28 @@
         content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
   <title>Product cards</title>
   <link rel="stylesheet" href="./style.css">
+  <link href="https://fonts.googleapis.com/css?family=Roboto:400,500,700&display=swap" rel="stylesheet">
 </head>
 <body>
-  <h1>Product cards</h1>
+  <div class="card" data-qa="card">
+    <img src="./images/imac.jpeg" alt="monoblok" class="img-item">
+    <h2 class="info">Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)</h2>
+    <div class="code">Код товара: 195434</div>
+    <div class="stars_comments">
+      <div class="stars">
+        <div class="star active"></div>
+        <div class="star active"></div>
+        <div class="star active"></div>
+        <div class="star active"></div>
```

Comment:
![image](https://user-images.githubusercontent.com/38065883/75557625-0a4aad00-5a49-11ea-951f-279724da72a3.png)
This rule required in this task also